dev/com.ibm.ws.session.cache_fat/fat/src/com/ibm/ws/session/cache/fat/SessionCacheTimeoutTest.java

```diff
@@ -0,0 +1,61 @@
+/**
+ *
+ */
+package com.ibm.ws.session.cache.fat;
+
+import static org.junit.Assert.assertNotNull;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.FATServletClient;
+
+/**
+ * Tests related to Session Cache Timeouts, using a server with the following session settings:
+ * invalidationTimeout="1s"
```

Comment:
I think a timeout of 1s is too small, as we may not be able to reliably hit that window for the requests we need to be successful.  Lets do something longer like 5s.

Reply:
I pick 1s to keep the test short, but since the reaper thread is minimum 30s, that's what we're most likely to be waiting on anyways, so I'll increase to 5s.